### PR TITLE
Updating the repo so that it builds

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: src/Microsoft.AspNet.SignalR.ServiceBus/project.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
 {
     "version": "3.0.0-*",
     "dependencies": {
-        "Microsoft.AspNet.SignalR.Server": "3.0.0-*"
+        "Microsoft.AspNet.SignalR.Server": "3.0.0-*",
+        "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0-*"
```
Comment (davidfowl) [inline src/Microsoft.AspNet.SignalR.ServiceBus/project.json]: We removed this pattern.

File: src/Microsoft.AspNet.SignalR.ServiceBus/ServiceBusServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -4,24 +4,25 @@
...
     public static class RedisServiceCollectionExtensions
     {
-        public static IServiceCollection AddServiceBus(this IServiceCollection services, Action<ServiceBusScaleoutConfiguration> configureOptions = null)
+        public static IServiceCollection AddServiceBus(this IServiceCollection services, Action<ServiceBusScaleoutOptions> configureOptions = null)
         {
             return services.AddServiceBus(configuration: null, configureOptions: configureOptions);
         }
 
-        public static IServiceCollection AddServiceBus(this IServiceCollection services, IConfiguration configuration, Action<ServiceBusScaleoutConfiguration> configureOptions)
+        public static IServiceCollection AddServiceBus(this IServiceCollection services, IConfiguration configuration, Action<ServiceBusScaleoutOptions> configureOptions)
```
Comment (davidfowl) [inline src/Microsoft.AspNet.SignalR.ServiceBus/ServiceBusServiceCollectionExtensions.cs]: Remove the IConfiguration

Comment (moozzyk) [inline src/Microsoft.AspNet.SignalR.ServiceBus/ServiceBusServiceCollectionExtensions.cs]: OK

**Comment by moozzyk** [comment]
:up::calendar:

**Pull request merged**